elasticdl/python/worker/worker.py

```diff
@@ -576,6 +577,7 @@ def report_gradient_to_ps(self, grads):
         for ps_id in range(self._ps_num):
             req = reqs[ps_id]
             req.version = self._model_versions_from_ps[ps_id]
+            req.learning_rate = K.get_value(self._model.optimizer.lr)
```

Comment:
One question, do we need to propose a concept: `chief worker`? The learning rate is only pushed by the chief worker to PS.

I am not very sure, we could merge this PR first.

Reply:
The model version which the chief worker gets maybe not continuous. So, it cannot schedule the learning rate precisely. The learning rate may overlap if the workers get the same model version. Here, we may need a trade-off.